.github/workflows/tests.yaml

```diff
@@ -1,15 +1,36 @@
 name: Tests
-on: # Run on all pushes and pull requests
+on: # Run on all pushes and pull requests. Eventually can run certain jobs on commit.
   push:
     branches:
       - main
   pull_request:
+
 jobs:
-  build:
+  test-base:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.11"]
```

Comment:
it's already out of date, but we probably should update the readme to say `3.11+`: https://github.com/gridstatus/gridstatus/blob/cd3a0dd556edcae6442c15669984b10af443a642/README.md#L36-L37

Reply:
Also in `CONTRIBUTING.md`:

https://github.com/gridstatus/gridstatus/blob/4f0a209fd98368f6b90e46728383994f0a278736/CONTRIBUTING.md#L45

Reply:
I ran into some troubles with 3.12 initially as well. I didn't look into them too deeply and just moved to 3.11, but figure it's worth mentioning.

Reply:
updated